lib/inventory_refresh/persister.rb

```diff
@@ -5,14 +5,16 @@ class Persister
 
     attr_reader :manager, :collections
 
-    attr_accessor :refresh_state_uuid, :refresh_state_part_uuid, :total_parts, :sweep_scope, :retry_count, :retry_max
+    attr_accessor :refresh_state_uuid, :refresh_state_part_uuid, :refresh_time_tracking, :total_parts, :sweep_scope, :retry_count, :retry_max
```

Comment:
Can we just store each timestamp as an instance var like the rest of these attributes?  That way we don't need another object in the openapi spec to track it (or if you want to create a RefreshTimes object in openapi go for it)

Reply:
@agrare why do we need RefreshTimes object in openapi?

I wanted to avoid definition of tons of `*_time` variables and delegate them all from workflow

Reply:
Don't we need to pass these to the ingress-api from the collector over the ingress openapi spec?

Reply:
yes, they are passed without hash...it's not problem until `from_hash(to_hash)` you mentioned ^^^. So OK